### PR TITLE
Add basic test using Jest

### LINKED ISSUE #103: Set up tests using Jest
Include some basic tests only, using https://facebook.github.io/jest/.

### PR DESCRIPTION
**Pull request by wisn**
Closes #103

### CODE REVIEW

**Comment by blazeu** [comment]
`Fix` ~> `Closes`

Also include that in the commit body.

File: tests/lib/generate.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    expect(file).toBeTruthy()
+  })
+
+  it('should create an index.html in the planet directory', () => {
+    const path = `${__dirname}/../../out/planet/index.html`
+    const file = fs.existsSync(path)
+
+    expect(file).toBeTruthy()
+})
```
Comment (jayvdb) [inline tests/lib/generate.test.js]: indent level

File: tests/lib/gather.test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+describe('lib.gather', () => {
+  it('should create a data.min.json file', () => {
+    const path = `${__dirname}/../../out/data.min.json`
+    const file = fs.existsSync(path)
+
+    expect(file).toBeTruthy()
+  })
+})
+
+// location: tests/gather.test.js
```
Comment (jayvdb) [inline tests/lib/gather.test.js]: remove all

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Comment by blazeu** [comment]
Nope, Closes goes in the body, 2 lines after the message

https://coala.io/commit

File: tests/lib/gather.test.js (hunk 2 above)
Comment (wisn) [reply tests/lib/gather.test.js]: Removed.

File: tests/lib/generate.test.js (hunk 1 above)
Comment (wisn) [reply tests/lib/generate.test.js]: Fixed.

**Comment by wisn** [comment]
@blazeu Is it?

**Review by jayvdb — APPROVED** [review]

**Pull request merged**